### DISCUSSION TITLE
Displays the size description of the selected object

### DISCUSSION THREAD

**Discussion by HafizKurniaAji** [uxbox/uxbox#1243] [Ideas]
I would like to suggest adding a size description to the selected object when we select it. This speeds up figuring out the size of an object.

From like this now,
<img width="321" alt="selected-object" src="https://user-images.githubusercontent.com/34779993/134786860-d6df1ef5-a651-40f7-b570-fbc486ec1fa0.png">

Then might be like this.
<img width="284" alt="gambar" src="https://user-images.githubusercontent.com/34779993/134786916-d8c95ca3-294b-488c-b4e2-f4593ed6b77b.png">

Thanks, Penpot team.

**Comment by elhombretecla** [comment]
Hi @hafizkurniaaji :)

More element info (size, measures, distances...) is displayed if you hold "alt" once a group or element is selected.

Do you think this could help you?

![measures-info](https://user-images.githubusercontent.com/1045247/136147427-d290212d-a4bd-43c5-bf9b-0eaba61c14bb.jpg)

**Reply by HafizKurniaAji** [reply]
Thanks, you've made me realize that object sizes also appear instead of just the dashed lines I usually use to align objects.

This may be because the font size for the object size number is quite small or the coloring contrast is lacking, causing me to be slow to notice that the sizes are listed at the bottom side.

**Reply by elhombretecla** [reply]
Your're right :)

Sometimes this information is hard to read. We are going to review it in order to improve contrast and positioning.

[https://tree.taiga.io/project/penpot/issue/2144](https://tree.taiga.io/project/penpot/issue/2144)

Thank you again!
